src/main/java/jp/co/dwango/logback/SlackWebhookAppender.java

```diff
@@ -190,9 +180,22 @@ private void initScriptEngine() throws ScriptException {
         // define payload function
         StringBuilder function = new StringBuilder();
         function.append("function payload(event) {");
-        function.append(  "var color, emoji, level = event.getLevel().toString(),");
-        function.append(      "timestamp = formatTimestamp(event.getTimeStamp()),");
-        function.append(      "message = event.getFormattedMessage();");
+        function.append(  "var property = (function() {");
+        function.append(      "var lcvo = event.getLoggerContextVO(),");
+        function.append(          "properties = lcvo.getPropertyMap();");
+        function.append(      "return function(name) {");
+        function.append(        "if(!name && name !== 0) {");
+        function.append(          "return 'Property_HAS_NO_KEY';");
+        function.append(        "}");
+        function.append(        "name = '' + name;");
+        function.append(        "var value = properties.get(name);");
+        function.append(        "return value != null ? value : java.lang.System.getProperty(name);");
+        function.append(      "};");
+        function.append(    "})(), color, emoji,");
+        function.append(    "hostname = property('HOSTNAME'),");
```

Comment:
changed the hostname to use the [HOSTNAME property](https://logback.qos.ch/manual/configuration.html#hostname).